packages/react/index.css

```diff
@@ -0,0 +1,10 @@
+body {
+  font-size: 20px;
+  padding: 1rem;
```

Comment:
Why are we specifying padding on the body tag?

Reply:
Nevermind, I think this is just for the demo?

Reply:
Yes it is. I will still clean it up a bit though.